src/main/java/com/google/sps/servlets/RoutesStoring.java

```diff
@@ -27,24 +28,35 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import org.json.JSONArray;
 
 /** Servlet that process information about user's markers. */
 @SuppressWarnings("serial")
 @WebServlet("/storeRoute")
 public class RoutesStoring extends HttpServlet {
+  static class MarkerId {
```

Comment:
I'm not quite sure what for do you need this class. Is it so that gson could parse? I'm not sure, but may it be possible to use "gson.fromJson(requestBody, String.class);" for that purpose?